Dockerfile-rhel

```diff
@@ -3,7 +3,7 @@ MAINTAINER Datadog <package@datadoghq.com>
 
 ENV DOCKER_DD_AGENT=yes \
     DD_LOGS_STDOUT=yes \
-    AGENT_VERSION=5.26.0-1 \
+    AGENT_VERSION=5.32.7-1 \
```

Comment:
let's use 5.32.8 instead, since it's released now (unless there's a reason not to?)

Reply:
sure `5.32.8-1`